<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IndexScreen</class>
 <widget class="QMainWindow" name="IndexScreen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>IndexScreen</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="mainFrame">
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="backgroundLabel">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>800</width>
         <height>600</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="../files.qrc">:/image/images/backgrounds/index.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="startButton">
       <property name="geometry">
        <rect>
         <x>67</x>
         <y>387</y>
         <width>312</width>
         <height>69</height>
        </rect>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">
	background-color: transparent;
	border: 0;
</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../files.qrc">
         <normaloff>:/image/images/sentences/startButton.png</normaloff>:/image/images/sentences/startButton.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>306</width>
         <height>63</height>
        </size>
       </property>
      </widget>
      <widget class="QPushButton" name="finishButton">
       <property name="geometry">
        <rect>
         <x>147</x>
         <y>497</y>
         <width>312</width>
         <height>69</height>
        </rect>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
	background-color: transparent;
	border: 0;
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../files.qrc">
         <normaloff>:/image/images/sentences/finishButton.png</normaloff>:/image/images/sentences/finishButton.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>306</width>
         <height>63</height>
        </size>
       </property>
      </widget>
      <widget class="QPushButton" name="osanaButton1">
       <property name="geometry">
        <rect>
         <x>570</x>
         <y>70</y>
         <width>191</width>
         <height>351</height>
        </rect>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{background: transparent;}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="osanaButton2">
       <property name="geometry">
        <rect>
         <x>500</x>
         <y>390</y>
         <width>191</width>
         <height>211</height>
        </rect>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{background: transparent;}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="moodDisplayLabel">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>280</y>
         <width>400</width>
         <height>60</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="../files.qrc">:/image/images/sentences/mood_normal.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="moodUpButton">
       <property name="geometry">
        <rect>
         <x>440</x>
         <y>275</y>
         <width>35</width>
         <height>35</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>MS UI Gothic</family>
         <pointsize>28</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 28pt &quot;MS UI Gothic&quot;;
color: red;
background-color: transparent;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../files.qrc">
         <normaloff>:/image/images/icons/smallUpButton.png</normaloff>:/image/images/icons/smallUpButton.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>30</width>
         <height>27</height>
        </size>
       </property>
      </widget>
      <widget class="QPushButton" name="moodDownButton">
       <property name="geometry">
        <rect>
         <x>440</x>
         <y>305</y>
         <width>35</width>
         <height>35</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>MS UI Gothic</family>
         <pointsize>28</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 28pt &quot;MS UI Gothic&quot;;
color: blue;
background-color: transparent;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../files.qrc">
         <normaloff>:/image/images/icons/smallDownButton.png</normaloff>:/image/images/icons/smallDownButton.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>30</width>
         <height>27</height>
        </size>
       </property>
      </widget>
      <widget class="QLabel" name="moodUpLabel">
       <property name="geometry">
        <rect>
         <x>460</x>
         <y>240</y>
         <width>70</width>
         <height>50</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 75 26pt &quot;UD デジタル 教科書体 N-B&quot;;
color: rgba(255, 169, 0, 1);</string>
       </property>
       <property name="text">
        <string>+10</string>
       </property>
      </widget>
      <widget class="QLabel" name="moodDownLabel">
       <property name="geometry">
        <rect>
         <x>460</x>
         <y>330</y>
         <width>70</width>
         <height>50</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 75 26pt &quot;UD デジタル 教科書体 N-B&quot;;
color: rgba(18, 93, 152, 1);</string>
       </property>
       <property name="text">
        <string>-10</string>
       </property>
      </widget>
      <widget class="QProgressBar" name="moodParameterBar">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>327</y>
         <width>401</width>
         <height>13</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QProgressBar {
    background-color: qlineargradient(spread:pad, x1:0, y1:0.511364, x2:1, y2:0.523, stop:0 #f64f59, stop:0.5 #c471ed ,stop:1 #12c2e9);
	color: rgb(200, 200, 200);
	border-style: none;
}
QProgressBar::chunk{
	background-color: rgb(98, 114, 164);
}</string>
       </property>
       <property name="value">
        <number>32</number>
       </property>
       <property name="textVisible">
        <bool>false</bool>
       </property>
       <property name="invertedAppearance">
        <bool>true</bool>
       </property>
       <property name="format">
        <string>%p</string>
       </property>
      </widget>
      <widget class="QLabel" name="moodLabel">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>257</y>
         <width>482</width>
         <height>100</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel#moodLabel {
	background-color: rgba(0, 0, 0, .2);
	border-radius: 15px;
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="logoLabel">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>30</y>
         <width>475</width>
         <height>170</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="../files.qrc">:/image/images/logos/title_logo.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
      <zorder>backgroundLabel</zorder>
      <zorder>moodLabel</zorder>
      <zorder>startButton</zorder>
      <zorder>finishButton</zorder>
      <zorder>osanaButton1</zorder>
      <zorder>osanaButton2</zorder>
      <zorder>moodDisplayLabel</zorder>
      <zorder>moodUpButton</zorder>
      <zorder>moodDownButton</zorder>
      <zorder>moodUpLabel</zorder>
      <zorder>moodDownLabel</zorder>
      <zorder>moodParameterBar</zorder>
      <zorder>logoLabel</zorder>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../files.qrc"/>
  <include location="../files.qrc"/>
 </resources>
 <connections/>
</ui>
